### PR TITLE
Notebook which demos annotation of Dense point clouds using SageMaker Ground Truth 

### PR DESCRIPTION
**Pull request by vidyaravipati**
Description of changes:
Add sample notebooks to demonstrate how to dowsample dense point clouds (> 1M points) , annotate the point cloud and upsample the point cloud back to original size

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Authored-by: Isaac Privitera privisaa@amazon.com
Authored-by: Vidya Sagar Ravipati ravividy@amazon.com
Authored-by: Jeremy Feltracco jpfelt@amazon.com

### CODE REVIEW

File: ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,744 @@
+{
```
Comment (TEChopra1000) [inline ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Remove this ARN and replace with <code>&lt;&lt;ADD WORK TEAM ARN HERE&gt;&gt;</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Suggest updating the first "Important" statement to :</p><p> </p><p><code>IMPORTANT: Make sure this IAM role has one or more IAM policies with the permissions described above attached.</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>remove line break after "format,"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Suggest adding cell about this and giving customers some information about what you're creating here. Recommendation:</p><p> </p><p>"In the following cell, you generate an input manifest file. This file tells Ground Truth where 3D point cloud input data is located and contains metadata about your input data. To learn more, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-point-cloud-input-manifest.html" target="_blank">Create an Input Manifest File for a 3D Point Cloud Labeling Job</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Add description above this cell. Suggestion:</p><p> </p><p>"In the following cell, you generate a label category configuration file. This file contains the label categories workers use to label the 3D point cloud, worker instructions, and other information about your task. To learn more about all of the options you have when you create a label category configuration file, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-label-cat-config-attributes.html" target="_blank">Create a Labeling Category Configuration File with Label Category and Frame Attributes</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Add description above:</p><p> </p><p>"In the following cell you configure the labeling job request. To learn more about the parameters used to configure a labeling job, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-create-labeling-job-api.html" target="_blank">Create a Labeling Job (API)</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Add description above:</p><p> </p><p>"Run the following cell to see the worker portal URL of your private workforce. Use this URL to login to the worker portal to see your labeling tasks. See the <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-point-cloud-worker-instructions.html" target="_blank">3D point cloud worker instructions</a> to learn more about the feature you'll see in the worker UI."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Need to describe what the cell is doing about the cell. From the blog we can put this:</p><p> </p><p>"Now that we have the downsampled labels, we will train a K-NN classifier from SKLearn to predict the full dataset labels by treating our annotated points as training data and performing inference on the remainder of the unlabeled points in our full size point cloud. Note that you can tune the number of points used as well as the distance metric and weighting scheme to influence how label inference is performed. If you label a few tiles in the full-size dataset, you can use those labeled tiles as ground truth to evaluate the accuracy of the K-NN predictions. This accuracy metric can then be used for hyperparameter tuning of K-NN or to try different inference algorithms to reduce your number of misclassified points between object boundaries, resulting in the lowest possible in-sample error rate."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

File: ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb [hunk 2]
```diff
@@ -8,18 +8,9 @@
     "\n",
```
Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>I suggest adding more content here so this notebook can stand alone (not be reliant on the blog post), or adding a link to the blog post if you can get it ahead of time. </p><p> </p><p>If you do the former, you can copy and paste some of the introductory content from the blog post. Suggest covering:</p><ul><li>What metrics are used in the notebook</li><li>How you can create a labeling job using just those frame in which anomalies were detected to audit and correct those labels.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>Notes:</p><ol><li>Tell customers to, if using a private workforce, replace <code>&lt;&lt;ADD WORK TEAM ARN HERE&gt;&gt;</code>  with the work team ARN in the cell below so they don't miss that step. </li><li>Add period after "Create and Manage Workforces"</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>Nit: remove "please" - this is not AWS style.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>Suggest modifying print statement to:</p><p> </p><p><code>IMPORTANT: Make sure this IAM role has one or more IAM policies attached that grant the permissions described above.</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>Should we link to the Object tracking specific part of the output data docs in this link at the end? https://docs.aws.amazon.com/sagemaker/latest/dg/sms-data-output.html#sms-output-video-object-tracking </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>It looks like "If" has an errant space between I and f and "there" has one between t and here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>Spacing issue after "The"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>Spacing issue after "experimental"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>spacing issue --&gt; "these" and "be  areas"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (TEChopra1000) [inline ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb:8]: <p>"SageMaker Ground truth" --&gt; "t" should be capitalized --&gt; "SageMaker Ground Truth"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

File: ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb (hunk 1 above)
Comment (vidyaravipati) [reply ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Ack</p>

Comment (vidyaravipati) [reply ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Updated the notebook</p>

Comment (vidyaravipati) [reply ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Updated it</p>

Comment (vidyaravipati) [reply ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Updated it</p>

Comment (vidyaravipati) [reply ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Updated it</p>

Comment (vidyaravipati) [reply ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb:1]: <p>Updated it</p>

File: ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb (hunk 2 above)
Comment (vidyaravipati) [reply ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb]: This notebook is not intended for review. Mistake in commit.Reverting the commit

Comment (vidyaravipati) [reply ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb]: This notebook is not intended for review. Mistake in commit.Reverting the commit

**Comment by vidyaravipati** [comment]
Closing this PR as I didn't realize this commit resulted in merging changes in Video quality annotation notebook
